org.otter_browser.OtterBrowser.json

```diff
@@ -0,0 +1,36 @@
+{
+  "app-id": "org.otter_browser.OtterBrowser",
```

Comment:
Are we still replacing dashes with underscores in IDs?

Reply:
Yes.